tools/core_packages.txt

```diff
@@ -46,6 +46,7 @@ p11kit
 pcre
 pcre2
 perl
+pixz
```

Comment:
Should we be setting `CREW_USE_PIXZ = 1` in `lib/const.rb`?

Reply:
I was thinking it might make sense to invert the logic and use PIXZ unless `CREW_USE_PIXZ=0`?

Reply:
That would work too.